SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -232,11 +232,13 @@ Stethoscope {
 	}
 
 	run {
-		synth.play(maxBufSize, bus, cycle);
-		if( view.notNil && synth.bufferIndex.notNil) {
-			scopeView.bufnum = synth.bufferIndex;
-			scopeView.start;
-		};
+		defer {
```

Comment:
same here?

Reply:
yes, because this can also cause problems when called from inside a task.
as the now-deferred methods all return the object, the interface remains the same.

Reply:
isn't this a method of a GUI object ? everyone should expect that such a call needs to be deferred ? i hesitate to add convenience behaviour because then you never know when you need it and when not  ...